src/EnergyPlus/ConvectionCoefficients.cc

```diff
@@ -180,39 +180,40 @@ void InitIntConvCoeff(EnergyPlusData &state,
-        state.dataConvectionCoefficient->GetUserSuppliedConvectionCoeffs = false;
+        state.dataConvect->GetUserSuppliedConvectionCoeffs = false;
     }
 
-    if (state.dataConvectionCoefficient->NodeCheck) { // done once when conditions are ready...
+    if (state.dataConvect->NodeCheck) { // done once when conditions are ready...
         if (!state.dataGlobal->SysSizingCalc && !state.dataGlobal->ZoneSizingCalc && state.dataZoneEquip->ZoneEquipInputsFilled &&
             allocated(state.dataLoopNodes->Node)) {
-            state.dataConvectionCoefficient->NodeCheck = false;
+            state.dataConvect->NodeCheck = false;
             for (int ZoneNum = 1; ZoneNum <= state.dataGlobal->NumOfZones; ++ZoneNum) {
-                if (Zone(ZoneNum).IntConvAlgo != HcInt::CeilingDiffuser) continue;
-                if (Zone(ZoneNum).SystemZoneNodeNumber != 0) continue;
+                    auto &zone = state.dataHeatBal->Zone(ZoneNum);
+                if (zone.IntConvAlgo != HcInt::CeilingDiffuser) continue;
+                if (zone.SystemZoneNodeNumber != 0) continue;
                 ShowSevereError(state,
                                 format("InitInteriorConvectionCoeffs: Inside Convection=CeilingDiffuser, but no system inlet node defined, Zone={}",
-                                       Zone(ZoneNum).Name));
+                                       zone.Name));
                 ShowContinueError(
-                    state, format("Defaulting inside convection to TARP. Check ZoneHVAC:EquipmentConnections for Zone={}", Zone(ZoneNum).Name));
-                state.dataHeatBal->Zone(ZoneNum).IntConvAlgo = HcInt::ASHRAETARP;
+                    state, format("Defaulting inside convection to TARP. Check ZoneHVAC:EquipmentConnections for Zone={}", zone.Name));
+                zone.IntConvAlgo = HcInt::ASHRAETARP;
             }
             // insert one-time setup for adaptive inside face
         }
     }
 
-    if (state.dataConvectionCoefficient->ActiveSurfaceCheck && !state.dataGlobal->SysSizingCalc && !state.dataGlobal->ZoneSizingCalc &&
+    if (state.dataConvect->ActiveSurfaceCheck && !state.dataGlobal->SysSizingCalc && !state.dataGlobal->ZoneSizingCalc &&
         state.dataZoneEquip->ZoneEquipSimulatedOnce) {
         SetupAdaptiveConvRadiantSurfaceData(state);
-        state.dataConvectionCoefficient->ActiveSurfaceCheck = false;
+        state.dataConvect->ActiveSurfaceCheck = false;
     }
 
-    if (state.dataGlobal->BeginEnvrnFlag && state.dataConvectionCoefficient->MyEnvirnFlag) {
+    if (state.dataGlobal->BeginEnvrnFlag && state.dataConvect->MyEnvirnFlag) {
         bool anyAdaptiveConvectionAlgorithm = false;
         for (int SurfNum = 1; SurfNum <= state.dataSurface->TotSurfaces; ++SurfNum) {
-            if (state.dataSurface->SurfIntConvCoeff(SurfNum) == HcInt::AdaptiveConvectionAlgorithm) {
+            if (state.dataSurface->surfIntConv(SurfNum).coeff == HcInt::AdaptiveConvectionAlgorithm) {
```

Comment:
Surface convection related fields were collected into two structs, one for interior convection and one for exterior.

Reply:
So this is a type. coeffType?

Reply:
MJW suggested that this be renamed from `coeff` to `model` and so I have a follow-on commit that does that and renames all of the related `convCoeff` to `convModel`. Does that work?

Reply:
Go with @mjwitte until I can prove otherwise, which is very rare.